src/components/TextField/TextField.tsx

```diff
@@ -22,19 +22,17 @@ export const TextField: React.FC<Props> = ({
   onChange = () => { },
   validate = () => true,
```

Comment:
```suggestion
  onChange,
  validate,
```

Make onChange required, don't set it to an empty function.
For validate - look next comment.